src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,11 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+<<<<<<< Updated upstream
+=======
+            <class name="Homework16"/>
+            <class name="Homework17"/>
+>>>>>>> Stashed changes
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210506030-b58df318-8528-448f-9f3d-00a7de9f0735.png)

For example 

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="LoginTests"/>
            <class name="BaseTest"/>
            <class name="Homework17"/>
        </classes>
    </test>
</suite>
```